src/app/explorePools/page.tsx

```
@@ -51,6 +93,103 @@ const ExploreFatePools = () => {
+
+        setPools(processedPools.filter(Boolean) as Pool[]);
+      } catch (error) {
+        console.error("Error fetching pools:", error);
+        setError("Failed to fetch pools. Please try again.");
+      } finally {
+        setLoading(false);
+      }
+    };
+
+    fetchPools();
+  }, [account?.address]); */}
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)